pkg/services/ngalert/models/instance_test.go

```diff
@@ -53,41 +53,225 @@ func buildTestInstanceStateTypeIsValidName(instanceType InstanceStateType, expec
 	return fmt.Sprintf("%q should not be valid", instanceType)
 }
 
+func TestInstanceCauseType_IsValid(t *testing.T) {
+	testCases := []struct {
+		instanceCauseType InstanceCauseType
+		expectedValidity  bool
```

Comment:
It's idiomatic just to use `expected` in tests. Not super important to change, but would be nice as it would be consistent 🙂